sites/public/__tests__/components/listing/GetApplication.test.tsx

```diff
@@ -0,0 +1,117 @@
+      <GetApplication
+        onlineApplicationURL={"online-app-url"}
+        applicationsOpen={true}
+        applicationsOpenDate={"November 20th, 2021"}
+        paperApplications={[
+          { fileURL: "file-url-en", languageString: "English" },
+          { fileURL: "file-url-es", languageString: "Spanish" },
+        ]}
+        paperMethod={true}
+        postmarkedApplicationsReceivedByDate={"November 30th, 2021"}
+        applicationPickUpAddressOfficeHours={"M-F 9am-5pm"}
+        applicationPickUpAddress={{
+          city: "City",
+          state: "State",
+          street2: "Street 2",
+          street: "Pick Up Address Street",
+          zipCode: "90210",
+        }}
+        preview={false}
+      />
+    )
+    expect(getByText("How to Apply")).toBeTruthy()
+    expect(getByText("Apply Online")).toBeTruthy()
+    expect(getByText("Apply Online").closest("a")?.getAttribute("href")).toBe("online-app-url")
+    expect(getAllByText("or").length).toBe(2)
+    expect(getByText("Get a Paper Application")).toBeTruthy()
+    expect(getByText("Download Application")).toBeTruthy()
+    expect(getByText("Pick up an application")).toBeTruthy()
+    expect(getByText("Pick Up Address Street", { exact: false })).toBeTruthy()
+    expect(getByText("Office Hours")).toBeTruthy()
+  })
+  it("do not render section when there is no paper application files and paper method is true", () => {
+    const { queryByTestId } = render(
+      <GetApplication
+        applicationsOpen={true}
+        applicationsOpenDate={"November 20th, 2021"}
+        paperApplications={[]}
+        paperMethod={true}
+        postmarkedApplicationsReceivedByDate={"November 30th, 2021"}
+        applicationPickUpAddressOfficeHours={"M-F 9am-5pm"}
+        applicationPickUpAddress={{
+          city: "City",
+          state: "State",
+          street2: "Street 2",
+          street: "Pick Up Address Street",
+          zipCode: "90210",
+        }}
+        preview={false}
+      />
+    )
+    expect(queryByTestId("get-application-section")).toBeNull()
+  })
+  it("disables buttons in preview state", () => {
+    const { getByText } = render(
+      <GetApplication
+        onlineApplicationURL={"online-app-url"}
+        applicationsOpen={true}
+        applicationsOpenDate={"November 20th, 2021"}
+        paperApplications={[
+          { fileURL: "file-url-en", languageString: "English" },
+          { fileURL: "file-url-es", languageString: "Spanish" },
+        ]}
+        paperMethod={true}
+        postmarkedApplicationsReceivedByDate={"November 30th, 2021"}
+        applicationPickUpAddressOfficeHours={"M-F 9am-5pm"}
+        applicationPickUpAddress={{
+          city: "City",
+          state: "State",
+          street2: "Street 2",
+          street: "Pick Up Address Street",
+          zipCode: "90210",
+        }}
+        preview={true}
+      />
+    )
+    expect(getByText("Apply Online").closest("button")?.disabled).toBe(true)
```

Comment:
This one as well could be `expect(getByRole("button", { name: "Apply Online" })).toBeDisabled()`. But no need to change for this issue